samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -326,8 +326,10 @@ - (UIImage*)transformImage:(UIImage*)image {
     // create CGImageRef from vImage_Buffer output
     CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
 
-    CGContextRef ctx = CGBitmapContextCreate(
-        outBuffer.data, outBuffer.width, outBuffer.height, 8, outBuffer.rowBytes, colorSpace, (CGBitmapInfo)kCGImageAlphaNoneSkipLast);
+    const CGBitmapInfo bitmapInfo = kCGImageAlphaNoneSkipLast | kCGBitmapByteOrder32Little;
```

Comment:
In this scenario we got lucky because there were multiple bugs which resulted in the misuse of bitmapInfo working correctly but purely by accident:
- CGBitmapContextCreate ignored byteOrdering so it didn't matter whether it was set correctly
- RGB Colorspaces were incorrectly coupled with the format BGRA and interpreted as such. So by specifying any RGB colorspace in CGBitmapContextCreate and any byteOrder value, BGRA was automatically implied no matter what. Since in the app is not using alpha and it really wants BGRX anyway, BGRA would operate almost identically to what it would expect it to.